parquet-column/src/main/java/org/apache/parquet/filter2/predicate/ValidTypeMap.java

```diff
@@ -74,9 +74,22 @@ private static void add(Class<?> c, FullTypeDescriptor f) {
     // Both of these binary types are valid
     add(Binary.class, new FullTypeDescriptor(PrimitiveTypeName.BINARY, null));
     add(Binary.class, new FullTypeDescriptor(PrimitiveTypeName.FIXED_LEN_BYTE_ARRAY, null));
+    add(Binary.class, new FullTypeDescriptor(PrimitiveTypeName.INT96, null));
```

Comment:
Add this here is just to make filter checking pass if user needs INT96 for a filter.
It seems INT96 is designed as Binary in PrimitiveType. Maybe higher level API could bind it with other java class. Not sure which to use here other than Binary?